Pages/Signup/styles.css

```diff
@@ -0,0 +1,33 @@
+@import url('/root.css');
+
+.form-signup{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 25rem;
+    height: 24rem;
+    background-color: var(--white-color);
+    margin: 5rem auto;
+    vertical-align:middle;
+    padding: 1rem 2rem;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
```

Comment:
If you are using this box-shadow color anywhere else, use a variable for the color